Task-5/dbconnection/connection.go

```diff
@@ -0,0 +1,38 @@
+package dbconnection
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+
+	"github.com/joho/godotenv"
+	_ "github.com/lib/pq"
+)
+
+// Database is exported to perform CRUD operations
+var Database *sql.DB
```

Comment:
don't make this global create struct in main.go
App{
 database *sql.DB
}

use inside api.